app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CommentsController < ApplicationController
+  def create
+    @current_user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(user_id: @current_user.id, post_id: @post.id, text: params[:comment])
+    return unless @comment.save
+
+    redirect_to "/users/#{@current_user.id}/posts/#{@post.id}"
```

Comment:
- [ ] **[Optional]** Great progress! Functionally, First users can create comments successfully. However, there's a slight issue with the redirect URL. For instance, if a User with ID `2` has a post with ID `4`, and the current user with ID `1` submits a comment, the redirect is currently set to `/users/1/posts/4`. It should ideally redirect to `/users/2/posts/4`, as User 2 owns the post.
To address this, consider updating the redirect URL logic to ensure it correctly redirects to the post's owner. 

**check out the video:**

https://github.com/ZabihullahNooriWardak/Blog-App/assets/55840999/299298a8-4af6-40ee-9adc-cb222e947788


```suggestion
    redirect_to "/users/#{@post.author_id}/posts/#{@post.id}"
```
This adjustment will enhance the overall user experience. Keep up the good work! 👍🚀